libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts

```diff
@@ -1,4 +1,11 @@
 import invariant from "invariant";
+
+/**
+ * these are either "dead"/"burn" addresses OR "abandon" seed addresses.
+ * These addresses are PUBLIC addresses
+ * We use them for tests and also for dry-run estimations
+ * DO NOT USE AS RECIPIENT OR SIGN TRANSACTIONS INTO THEM
```

Comment:
Maybe in this comment -- and for my own knowledge -- you could document how to get an abandon address.
I'm curious to know how you got them for all the different coins.

Reply:
i'll provide an internal doc